src/saveload/saveload.h

```diff
@@ -497,7 +497,7 @@ enum SaveLoadType : byte {
 	SL_REF         =  1, ///< Save/load a reference.
 	SL_ARR         =  2, ///< Save/load an array.
 	SL_STR         =  3, ///< Save/load a string.
-	SL_LST         =  4, ///< Save/load a list.
+	SL_REFLST      =  4, ///< Save/load a reference list.
```

Comment:
Since you're renaming & not preserving length anyway, why not just go for the full `SL_REFLIST` ?

Reply:
I kept the "short" notation for all of them .. but if we don't mind tables being (even more) misaligned, yeah, lets go for more descriptive names.

But just so to check, it does mean that the SLE name will also be `SLE_REF_LIST`, and that tables align even harder. You okay with that?